### ISSUE TITLE
Log in Button does not do anything on iOS.

### ISSUE THREAD

**Issue by leggettc18** [nextcloud/talk-ios#226]
Hello,

We have a self hosted instance of Nextcloud that we wanted to use primarily for NextCloud Talk. It all works just fine through the Android App and through the Web Browser, but users are unable to log in through the NextCloud talk iOS app.

After they type in our server url, they get the page with the Log In button that would take them to the page where they type their username and password, and it is themed appropriately according to our custom logo and color. However clicking the Log In button does not do anything. It plays an animation when clicking but does not go anywhere. We have tried this on an iPhone 7+ and an iPhone X. Please let me know if there is any other information you need.

**Comment by Ivansss** [comment]
Hi @leggettc18,

Have you tried clicking the 'Log in' button twice? Sometimes, I had also experienced that issue that I saw the animation but nothing happened after clicking the button a second time, but it's hard to reproduce this behavior for me.

If that's not the case, you could send me your Nextcloud server url and some test account credentials to `ivan AT nextcloud DOT com` so we could debug it.

**Comment by leggettc18** [comment]
Ok, I just sent it to you, and realized I mistyped the URL and sent a correction email as well.

**Comment by jniehues** [comment]
Can confirm this behaviour with my self-hosted instance on iOS (iPhone 8) with Nextcloud Talk for iOS 6.1.1.
Issue seems to be iOS specific as my Android NC Talk App works flawless. My NC instance is reachable online and listening on the regular https-port 443.

**Comment by Ivansss** [comment]
Hi @jniehues,

We found some issues in the TLS configuration of @leggettc18 server.
I was waiting for some feedback to close this issue.

You can also send me and email with your Nextcloud server url and some test account credentials, so we can check what is your issue.
It could be that your server doesn't fulfill [ATS requirements](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html#//apple_ref/doc/uid/TP40009251-SW57)

**Comment by leggettc18** [comment]
Any ideas for things to try to fix the problem with our server? Our research thus far has not turned up much.

**Comment by jniehues** [comment]
@leggettc18 Do you run a reverse proxy for accessing your NC instance?

**Comment by jniehues** [comment]
Okay, I got it. The problem was an issue with the redirecting.
My reverse proxy is doing a redirecting from http to https. Checked the configuration because I thought this might be the cause. After not getting things fixed I've done more research on it. While accessing Nextcloud directly over https I was wondering about the redirecting from https to http and then once again back to https. After having a look at the rev. proxy log and the nextcloud container log, I figured out that there was an additional redirecting inside the container (apache2/nc).
Inside the Nextcloud config.php I finally found the cause. There was an option for redirecting 
`'overwrite.cli.url' => 'http://cloud.mydomain.com'` which I replaced with `'overwritehost' => 'cloud.mydomain.com'` followed by `'overwriteprotocol' => 'https'`.
After this the login inside the iOS-Talk app is working for me.

Thanks @Ivansss for the hint!
